src/main/kotlin/com/dp/advancedgunnerycontrol/weaponais/SpecificAIPluginBase.kt

```diff
@@ -212,37 +213,21 @@ abstract class SpecificAIPluginBase(
 
     protected open fun shouldConsiderNeutralsAsFriendlies(): Boolean = true
 
-    /**
-     * @return approximate angular distance of target from current weapon facing in rad
-     * note: approximation works well for small values and is off by a factor of PI/2 for 180°
-     * @param entity: Relative coordinates (velocity-compensated)
-     */
-    protected fun angularDistanceFromWeapon(entity: Vector2f): Float {
-        val weaponDirection = vectorFromAngleDeg(weapon.currAngle)
-        val distance = entity - weapon.location
-        val entityDirection = distance times_ (1f / distance.length())
-        return (weaponDirection - entityDirection).length()
-    }
-
     // if aimPoint == null, the current weapon facing will be used
     protected fun isFriendlyFire(friendlies: List<Pair<CombatEntityAPI, Vector2f>>, aimPoint: Vector2f? = null): Boolean {
-        return friendliesInDangerZone(friendlies, aimPoint).isNotEmpty()
+        return isAimable(weapon) && friendliesInDangerZone(friendlies, aimPoint).isNotEmpty()
```

Comment:
I added `isAimable(weapon)` here, so that missiles don't count towards friendly fire. Thay fly over friendles, so this shouldn't be a concern. Or are there some specific cases I missed?

Reply:
To be honest, I'm not 100% certain. Some missiles can't fly over/through allies, but I think all non-aim-able missiles should be able to. So let's just add this and assume that, unless someone complains, it's fine xD